app/(docs)/dcs/third-party-tools/truenas/page.md

```diff
@@ -135,6 +135,8 @@ In order to see the data uploaded to your bucket in the web console, you must un
 
 To complete the integration, you will need the S3 credentials created in the previous steps and a TrueNAS login.
 
+If you use TrueNAS 24.10 (Electric Eel) or later, you would be able to setup a TrueCloud Backup Task, it's a preferred method, because it has several advantages above the Cloud Sync Task: [https://www.truenas.com/docs/scale/24.10/scaletutorials/dataprotection/truecloudtasks/](https://www.truenas.com/docs/scale/24.10/scaletutorials/dataprotection/truecloudtasks/)
```

Comment:
```suggestion
If you use TrueNAS 24.10 (Electric Eel) or later, you would be able to setup a [TrueCloud Backup Task](#create-a-truecloud-backup-task), it's a preferred method, because it has several advantages above the [Cloud Sync Task](#create-a-cloud-sync-task): [https://www.truenas.com/docs/scale/24.10/scaletutorials/dataprotection/truecloudtasks/](https://www.truenas.com/docs/scale/24.10/scaletutorials/dataprotection/truecloudtasks/)
```